snuba/datasets/table_storage.py

```diff
@@ -157,20 +144,14 @@ def get_bulk_writer(
         This is a stripped down verison of the writer designed
         for better performance when loading data in bulk.
         """
-        # TODO: Consider using rapidjson to encode everywhere
-        # once we will be confident it is reliable enough.
-
         from snuba import settings
 
         table_name = table_name or self.__table_schema.get_table_name()
 
         options = self.__update_writer_options(options)
 
         return self.__cluster.get_writer(
-            table_name,
-            lambda row: rapidjson.dumps(row).encode("utf-8"),
-            options,
-            chunk_size=settings.BULK_CLICKHOUSE_BUFFER,
+            table_name, options, chunk_size=settings.BULK_CLICKHOUSE_BUFFER,
         )
```

Comment:
These methods will likely be consolidated at some point in the future but not in a hurry to do so now.